docs/requirements.in

```
@@ -4,3 +4,4 @@ Sphinx >= 4.1.2
 sphinx-ansible-theme >= 0.8.0
 sphinx-notfound-page >= 0.7.1
 sphinxcontrib-apidoc >= 0.3.0
+sphinx-copybutton
```

Set the lower bound:
```suggestion
sphinx-copybutton >= 0.4.0
```

Also, keep the order alphabetical.

But it's not all, you must now regenerate the `docs/requirements.in` file (the command is in its header comment).

Though, I'd strongly recommend integrating this extension in a separate PR so that it doesn't block the main effort of actually adding the content..